.github/workflows/run-integration-test.yml

```diff
@@ -20,14 +20,11 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./
         with:
-          command: "echo 'yarn vercel:cleanup && yarn vercel:deploy --local-config=vercel.customer1.staging.json and found https://app.vercel.app'"
+          command: "yarn test:integration"
+          apply_domain_aliases: true
+          failIfAliasNotLinked: false
         env:
-          VERCEL_TOKEN: "temporary nothing"
-      - uses: ./
-        with:
-          command: ls
-      - uses: ./
-        with:
-          command: echo "test"
-        env:
-          VERCEL_TOKEN: "temporary nothing"
\ No newline at end of file
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_DEPLOYMENT_URL: "https://nrn-v2-mst-aptd-at-lcz-sty-c1.vercel.app"
```

Comment:
Why providing `VERCEL_DEPLOYMENT_URL `?

Reply:
And why is that an url related to NRN? It shouldn't

Reply:
This is how works the integration tests. You hardcode a vercel url you own, so you can apply aliases on it.